### ISSUE TITLE
Bug - [Dropdown] - [Dropdown tooltips are not accessible by JAWS and NVDA]

### ISSUE THREAD

**Issue by jessiehuff** [patternfly/patternfly-react#7584]
When testing the tooltips on dropdown items, they seem to work with VO. However, I can not get them to work with JAWS or NVDA. It seems that @thatblindgeye was able to get it to work though so it might be worth exploring to see if others are able to reproduce. Perhaps it's a version issue or some other combination that we can narrow down, but I think we should look into it in to ensure this is accessible for a large portion of screen reader users.

**How do you reproduce the problem?**
Open the dropdown with a tooltip on JAWS or NVDA. Navigate by either using the contextual approach with "read next item" commands or by Tab approach. JAWS and NVDA simply announce the list item without the tooltip.
